Doc number of search results is not available with secure search (#4066)
quiltdata/quilt: docs/advanced-features/s3-prefix-permissions.md

* Prefix-aware search performs a head request on every object result; this may slow
search performance in the Catalog

> * Number of search results is not available with prefix-aware search

Reviewer: ```suggestion
* The Catalog does not (accurately) display the number of search results
```

Author: this limitation applies only to prefix-aware search aka secure search and you suggestion remove its mention
